official/resnet/resnet_run_loop.py

```diff
@@ -37,6 +38,8 @@
 from official.utils.logs import logger
 from official.utils.misc import distribution_utils
 from official.utils.misc import model_helpers
+from official.resnet.keras import resnet_model as keras_resnet_model
+from tensorflow.contrib.data.python.ops import threadpool
```

Comment:
Let's sync with @jsimsa about what the recommended way of doing this (if any) is, without having to directly import a `contrib` module.

Also, shouldn't this be `tensorflow.data.experimental`?

Reply:
Currently, the process of overriding the threadpool is not automated; the import should indeed be `tensorflow.data.experimental.python.ops`.

On a related note, I think we should get rid of this. AFAIK, the model garden is meant to provide idiomatic TensorFlow implementations and I would avoid including experimental APIs that might be need to for top of the line performance. If the top of the line performance is indeed needed, I would prefer to expose the threadpool override through a tf.data option.

Reply:
Sounds good. Will clarify this in person tomorrow.